ManageWikiExtensions.php

```diff
@@ -1945,7 +1945,7 @@
 			],
 		],
 		'snapprojectembed' => [
-			'name' => 'SnapProjectEmbed',
```

Comment:
On mw.org the ext page even refers to it as SnapProjectEmbed

Reply:
I am an upstream dev and can confirm I forgot to update it.

Reply:
@Pix1234, also see snapwiki/SnapProjectEmbed#12.